src/components/views/Inbox.vue

```diff
@@ -35,18 +35,17 @@ export default {
     StudentQuestionService.getStudentQuestions(this, {})
       .then(
         (questions) => {
-          for (let i=0; i<questions.length; i++) {
-
+          questions.map((q) => {
             let o = {};
```

Comment:
What is the overlap between the object `o` and the question `q`? It seems like you are constructing a very similar object. Use either `_.omit` (if there is stuff that you don't want from `q`) or object spread if you just want `q` (`{ ...q}`)

Also give variables clearer names. What is `o` in this?

Reply:
@stackptr Yeah, you're actually right, they're almost the same :thinking:. 

I guess that it ended up like this because in the beginning I thought that I only needed a couple of fields from the question object, but as I progressed I realized that I needed more fields, so I added them, but I was unaware that by the time I finished, I had already added almost all of the fields.

I refactored it to push the question without using a proxy object ([latest commit](https://github.com/UPchieve/web/pull/68/commits/8ccc5759fff9a251f73b97c6202436e33005573a)).